bats/fb-content-katello.bats

```diff
@@ -140,12 +140,21 @@ EOF
   subscription-manager repos --enable="${ORGANIZATION_LABEL}_${PRODUCT_LABEL}_${YUM_REPOSITORY_LABEL}" | grep -q "is enabled for this system"
 }
 
-@test "install katello-agent" {
-  tPackageInstall katello-agent && tPackageExists katello-agent
+@test "install katello-host-tools" {
+  tPackageInstall katello-host-tools && tPackageExists katello-host-tools
+}
+
+@test "install package locally" {
+  run yum -y remove walrus
```

Comment:
While it may not be an assertion I do think you want to be sure it's not installed before. Otherwise you're not testing what you think you're testing. I don't mind you merging it but in my experience this is the sort of thing that can mask real problems.